### DISCUSSION TITLE
403 Forbidden when calling ListIssuedDocuments with entity.vat_number as filter

### DISCUSSION THREAD

**Discussion by ghost** [fattureincloud/api#71] [Q&A] [answered]
Hi!

As the title says I keep getting 403 Forbidden when calling ListIssuedDocuments api with entity.vat_number as query parameter.
I tried replacing entity.vat_number with entity.name and everything works fine. Here is the code:

```vb.net
    Dim config As New Configuration()
    config.BasePath = basePath
    config.AccessToken = accessToken
    Try
        Dim filter As New Filter()
        filter.Where("entity.vat_number", [Operator].EQ, entityVatNumber)
        'filter.And("date", [Operator].EQ, [date].ToString("yyyy-MM-dd"))
        filter.And("number", [Operator].EQ, number)
        Dim q = filter.BuildQuery()

        Dim issuedDocumentsApi As New IssuedDocumentsApi(config)
        Dim listIssuedDocumentsResponse = issuedDocumentsApi.ListIssuedDocuments(companyId, "invoice", Nothing, "detailed", Nothing, Nothing, Nothing, q)
        If listIssuedDocumentsResponse.Data.Count > 0 Then
            Dim issuedDocument = listIssuedDocumentsResponse.Data.Find(Function(id) id.Date = [date])
            Return issuedDocument
        Else
            Return Nothing
        End If
        'Return If(listIssuedDocumentsResponse.Data.Count > 0, listIssuedDocumentsResponse.Data(0), Nothing)
    Catch ex As ApiException
        Return Nothing
    End Try
```

Any help would be much appreciated
Thanks!

**Comment by valmoz** [comment, marked as answer]
Hi Diego,
really strange.
Could you please contact our customer support, linking this discussion? I need to check some sensible data to verify your case and I don't want to publish it in this public discussion.

Thanks
Mauro

**Reply by ghost** [reply]
Hi @valmoz 
How can I do it? I'm just a developer so I need to forward this to the administration
Thanks!

EDIT: If it can help, searching a client by his vat_number works fine

**Reply by valmoz** [reply]
The support chat is on the Fatture in Cloud main page.
Otherwise, you can just share the company ID and timestamp of the error here and I'll delete the message (even if it isn't the best solution).

**Reply by ghost** [reply]
Thanks I'll write there then, can you tell me all needed data?

**Reply by valmoz** [reply]
As I said, we need just the companyID and the request timestamp.
Plus, you should attach this discussion link and write to the support that I was the one who suggested to write there.

Thanks
Mauro

**Reply by ghost** [reply]
Will do, thanks!

EDIT: Just did it

**Reply by valmoz** [reply]
Hi, @diegonogaretti 
were you able to solve the 403 issue?

Thanks
Mauro

**Reply by ghost** [reply]
Hi @valmoz 
Yes, after chatting with your support team, which asked us to try calling the same api via Postman, it turned out that (at least in our case) the following code adds '(' ')' to the query when multiple fields are used and that causes the 403 error. Removing the brackets immediatly gave us 200.
```vb.net
Dim filter As New Filter()
filter.Where("entity.vat_number", [Operator].EQ, entityVatNumber)
filter.And("number", [Operator].EQ, number)
Dim q = filter.BuildQuery()
```
Thanks!

EDIT: Support team told us that developers told them brackets shouldn't be a problem but in our case (with both SDK and Postman) they were giving 403

**Reply by valmoz** [reply]
Ok, thank you @diegonogaretti 
we'll investigate the issue with brackets.

If you have further questions, feel free to contact us.
Thanks 
Mauro
